CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 CHANGELOG
 =============
 
-## 0.8.0 (WIP)
+## 1.1.0 (WIP)
```

Comment:
as this won't be 1.1.x@dev branch in progress, this part in changelog is pointless in changelog file.

It could be added back when changing version to 1.1.x@dev... if really needed in changelog...